### ISSUE TITLE
PATCH: Origin line enhancements

### ISSUE THREAD

**Issue by rsbx** [leozide/leocad#826]
This patch

1. Adds the Z origin line
2. Fixes the Z-fighting of origin lines with the base grid
3. Extends the origin lines past the base grid  by 1 grid unit
 
[originlines.diff.txt](https://github.com/leozide/leocad/files/9635280/originlines.diff.txt)

**Comment by hendursaga** [comment]
@rsbx you new to GitHub? It would be best to have sent this as a Pull Request instead of a bug report.

**Comment by nathaneltitane** [comment]
> @rsbx you new to GitHub? It would be best to have sent this as a Pull Request instead of a bug report.

@hendursaga do you know how to polite?

**Comment by hendursaga** [comment]
@nathaneltitane Yes, I do, and I am. I might've needed to word that differently, but I was offering to help if requested versus letting it disappear down the memory hole.

**Comment by hendursaga** [comment]
@rsbx I went ahead and created a PR on your behalf, see https://github.com/leozide/leocad/pull/827. In the future, please do create a Pull Request so that it eases the process. The patch file DID apply without a hiccup, though!

**Comment by nathaneltitane** [comment]
@hendursaga still no pull from @leozide
